examples/ecs-ec2/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "ecs_cluster_name" {
+  description = "Name of the AWS ECS Cluster to deploy the Coralogix OTEL Collector. Supports Amazon EC2 instances only, not Fargate."
+  type        = string
+}
+
+variable "image_version" {
+  description = "The Coralogix Open Telemetry Distribution Image Version/Tag. Defaults to \"latest\". See: https://hub.docker.com/r/coralogixrepo/coralogix-otel-collector/tags"
```

Comment:
We don't use latest should be selected manually and we don't need to have default